src/worker.py

```diff
@@ -233,16 +257,6 @@ def stop(self) -> None:
         self.manager.stop()
         self.oak.close()
 
-    def _add_msgs_from_queue(
```

Comment:
Just changing location of this fn?

Reply:
Yeah changing the call to be within `get_most_recent_msg`